deps/base64/base64/lib/config.h

```diff
@@ -1 +1,7 @@
-// Intentionally empty
```

Comment:
This should remain empty

Reply:
Accepted.

Reply:
I meant you need to also revert the other changes to this file, otherwise compilation will fail on other platforms that do not have the features being defined here.